app/src/main/play/contactEmail

```diff
@@ -0,0 +1 @@
+googleplay@nutomic.com
```

Comment:
Is this intended?

Reply:
OK. If you get no e-mails, then you have got a good spam filter (otherwise one gets multiple advertisements for marketing companies when a mail address is put on Google Play)